### PR TITLE
update sharing for screening

### PR DESCRIPTION
**Pull request by peggy-quartech**
# Description

This PR includes the following proposed change(s):

- https://justice.gov.bc.ca/jira/browse/SPDBT-2629

### CODE REVIEW

File: src/Spd.Resource.Repository.IntegrationTest/ApplicationRepositoryTest.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        clearance.spd_workswith = (int)Enum.Parse<WorksWithChildrenOptionSet>(existingClearanceWorkWith.ToString());
+        clearance.spd_clearanceid = Guid.NewGuid();
+        clearance.spd_dategranted = DateTime.Now.AddDays(-200);
+        //clearance.spd_expirydate = DateTimeOffset.UtcNow.AddDays(200); //dynamics automatically set it based on dategranted
+        clearance.spd_sharable = (int)YesNoOptionSet.Yes;
+        _context.AddTospd_clearances(clearance);
+        _context.SetLink(clearance, nameof(clearance.spd_ServiceType), _context.LookupServiceType(existingClearanceServiceType.ToString()));
+        _context.SetLink(clearance, nameof(clearance.spd_ContactID), contact);
+        await _context.SaveChangesAsync();
+
```
Comment (ruaraya) [inline src/Spd.Resource.Repository.IntegrationTest/ApplicationRepositoryTest.cs:56]: Please remove additional line break

File: src/Spd.Resource.Repository.IntegrationTest/ApplicationRepositoryTest.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        // Assert
+        Assert.NotNull(resp);
+        Assert.Equal(expectedClearanceCount, resp.Clearances.Count());
+
+        //Annihilate
+        _context.DeleteObject(clearance);
+        _context.DeleteObject(contact);
+        await _context.SaveChangesAsync();
+    }
+
```
Comment (ruaraya) [inline src/Spd.Resource.Repository.IntegrationTest/ApplicationRepositoryTest.cs:77]: Please remove additional line break

File: src/Spd.Resource.Repository.IntegrationTest/ApplicationRepositoryTest.cs (hunk 1 above)
Comment (peggy-quartech) [reply src/Spd.Resource.Repository.IntegrationTest/ApplicationRepositoryTest.cs:56]: sure

File: src/Spd.Resource.Repository.IntegrationTest/ApplicationRepositoryTest.cs (hunk 2 above)
Comment (peggy-quartech) [reply src/Spd.Resource.Repository.IntegrationTest/ApplicationRepositoryTest.cs]: fixed

**Review by ruaraya — APPROVED** [review]

**Pull request merged**